.github/workflows/deploy-infrastructure.yml

```diff
@@ -135,12 +142,17 @@ jobs:
         run: sudo snap install --classic aws-cli
       - name: Check if any deployments are running
         run: ../scripts/check-for-running-deployments.sh ${{ inputs.environment }}
+      - name: Set Mavis app version
+        run: |
+          GIT_REF_TO_DEPLOY="${{ inputs.git_ref_to_deploy }}"
+          echo "GIT_REF_TO_DEPLOY=$GIT_REF_TO_DEPLOY" >> $GITHUB_ENV
```

Comment:
I don't think we're using this environment variable anywhere, only `MAVIS_APP_VERSION`.